docs/import/center.js

```diff
@@ -20,37 +19,37 @@ function withGeo(feature) {
  * @param {ee.FeatureCollection} features the featureCollection around which you
  * want to orient the map
  * @param {Promise} firebaseAuthPromise
+ * @return {Promise<Array<number>>} Promise that completes with array of bounds
+ * after Firestore write is complete.
  */
 function storeCenter(features, firebaseAuthPromise) {
   const damageWithCoords = ee.FeatureCollection(features.map(withGeo));
   // This is assuming we're not crossing the international date line...
   const outerBounds = ee.List([
-    damageWithCoords.aggregate_max('lat'),
-    damageWithCoords.aggregate_max('lng'),
     damageWithCoords.aggregate_min('lat'),
     damageWithCoords.aggregate_min('lng'),
+    damageWithCoords.aggregate_max('lat'),
```

Comment:
why flipped?

Reply:
Because I think people generally process bounds as (min, max), and especially for LTR-native speakers, the left-most bound should come first. So, lower-left-to-upper-right is more natural than the reverse. Switched because this array is being displayed to users now.